examples/v2/config_chat2.nim

```diff
@@ -225,6 +225,11 @@ type
       desc: "WebSocket listening port."
       defaultValue: 8000
       name: "websocket-port" }: Port
+    
+    websocketSecureSupport* {.
+      desc: "WebSocket Secure Support."
+      defaultValue: false
+      name: "websocket-Secure-Support" }: bool
```

Comment:
```suggestion
      name: "websocket-secure-support" }: bool
```

Reply:
We can probably change the websocket enable/disable config items to simply `websocket` and `websocket-secure`?